ctlib.pc.in

```diff
@@ -0,0 +1,21 @@
+# 
+#  pkg_config.pc.in
+#
+#  pkg_config configuration file 
+#  For a detailed description of options, please visit
+#  Dan Nicholson’s Guide to pkg-config (http://www.freedesktop.org/wiki/Software/pkg-config/)
+#
+
+prefix="@CMAKE_INSTALL_PREFIX@"
+includedir=${prefix}/include
+libdir=${prefix}/lib/
+
+Name: CT-library
+Version: @VERSION@
+Description: A free implementation of TDS protocol
+URL: https://github.com/FreeTDS/freetds
+Requires: 
+Requires.private: libssl, libcrypto libiconv
+Cflags: -I${includedir}/freetds

```

Comment:
files are installed in `${includedir}`, not `${includedir}/freetds`. Not sure how MingW package this... maybe it would be better to use `includedir="@CMAKE_INSTALL_INCLUDEDIR@"` above ?